.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,46 @@
+name: Markdownlint
+
+on:
+  pull_request:
+    branches-ignore:
+      - translations
+  push:
+    branches:
+      - main
+
+jobs:
+  see_if_should_skip:
+    runs-on: ubuntu-latest
+
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@36feb0d8d062137530c2e00bd278d138fe191289
+        with:
+          cancel_others: 'false'
+          github_token: ${{ github.token }}
+          paths: '["**/*.md", "!translations/**", "package*.json", ".markdownlint.json", ".github/workflows/markdownlint.yml", ".github/workflows/markdownlint-problem-matcher.json"]'
+          paths_ignore: '["translations/**"]'
+  lint:
```

Comment:
Can we rename this to something other than "lint" to avoid confusion with the JavaScript-linting job? Maybe `mdlint` or `markdown-lint`? 🤷‍♂️

Reply:
Figured it was better to use the same job name so it stays a required check